deployment/cloud-run/secure_api_server.py

```
@@ -479,34 +530,488 @@
+            'pipeline_status': pipeline_status
+        }
+
+
 def initialize_model():
     """Initialize the emotion detection model"""
+    global T5_AVAILABLE, WHISPER_AVAILABLE
+
     try:
         logger.info("🚀 Initializing emotion detection API server...")
         logger.info(f"📊 Configuration: MAX_INPUT_LENGTH={MAX_INPUT_LENGTH}, RATE_LIMIT={RATE_LIMIT_PER_MINUTE}/min")
         logger.info(f"🔐 Security: API key protection enabled, Admin API key configured")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)